### PR TITLE
DOMA-5906 extend postmessage analytics event name validator list

### PR DESCRIPTION
**Pull request by sitozzz**

### CODE REVIEW

File: apps/condo/domains/common/components/PostMessageProvider/validators.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@ const NoParamsValidator = ajv.compile(NoParamsSchema)
 const CondoWebSendAnalyticsEventParamsSchema = {
     type: 'object',
     properties: {
-        event: { type: 'string', enum: ['click', 'check'] },
+        event: { type: 'string', enum: ['click', 'check', 'change', 'select'] },
```
Comment (SavelevMatthew) [inline apps/condo/domains/common/components/PostMessageProvider/validators.ts:19]: How about removing select to change btw if you're touching analytics btw?)

Comment (sitozzz) [reply apps/condo/domains/common/components/PostMessageProvider/validators.ts]: Done that

File: packages/ui/src/components/Select/select.tsx [hunk 2]
```diff
@@ -66,8 +66,7 @@ const Select = <ValueType extends SelectValueTypeBase>(props: SelectProps<ValueT
         }
 
         if (title) {
-            // TODO(DOMA-5597): Move it to change event; Also key should be used as well
```
Comment (SavelevMatthew) [inline packages/ui/src/components/Select/select.tsx:69]: left a todo about key and multiple select options please, it's still not fixed :)

**Review by SavelevMatthew — APPROVED** [review]

File: packages/ui/src/components/Select/select.tsx (hunk 2 above)
Comment (sitozzz) [reply packages/ui/src/components/Select/select.tsx:69]: Done

**Pull request merged**